src/tt_citites_skylines/exporter_dae.rb

```diff
@@ -0,0 +1,109 @@
+#-------------------------------------------------------------------------------
+#
+# Thomas Thomassen
+# thomas[at]thomthom[dot]net
+#
+#-------------------------------------------------------------------------------
+
+module TT::Plugins::CitiesSkylinesTools
+
+  def self.export_asset_dae
+    # Find the destination for the asset.
+    default_path = self.find_asset_path
+    filename = self.get_model_dae_name
+    if Sketchup.platform == :platform_win
+      dae_filter = "COLLADA File (*.dae)|*.dae||"
```

Comment:
i just copied your exporter, removed unneeded stuff and added the translation code. should be merged into one file i guess.